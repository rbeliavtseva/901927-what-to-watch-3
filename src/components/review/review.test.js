import React from "react";
import renderer from "react-test-renderer";
import Review from "../review/review.jsx";

const review = {
  name: `Kate Middleton`,
  text: `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
        Placerat vestibulum lectus mauris ultrices. Ultrices dui sapien eget mi proin. Vitae auctor eu augue ut. Parturient montes nascetur ridiculus mus mauris vitae.`,
  date: `April 1, 2016`,
  rating: `10,0`
};

it(`Render Review`, () => {
  const tree = renderer
    .create(
        <Review
          review={review}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
