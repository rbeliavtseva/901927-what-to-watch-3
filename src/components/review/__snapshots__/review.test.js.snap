// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Review 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
        Placerat vestibulum lectus mauris ultrices. Ultrices dui sapien eget mi proin. Vitae auctor eu augue ut. Parturient montes nascetur ridiculus mus mauris vitae.
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Kate Middleton
      </cite>
      <time
        className="review__date"
        dateTime="2016-12-24"
      >
        April 1, 2016
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    10,0
  </div>
</div>
`;
